parsers/to-react-native/to-react-native.spec.ts

```diff
@@ -15,6 +15,15 @@ describe('To React Native', () => {
     expect(await toReactNative(tokens, { assetsFolderPath: '../path' }, libs)).toMatchSnapshot();
     expect(await toReactNative(tokens, { assetsFolderPath: '../path/' }, libs)).toMatchSnapshot();
   });
+  it('Should support the typescript.castToConst option', async () => {
+    const tokens = seeds().tokens;
+    const result = await toReactNative(
+      tokens,
+      { assetsFolderPath: 'path', typescript: { castToConst: true } },
+      libs,
+    );
```

Comment:
We should afford an extra check for the string we expect to find in this case.
```suggestion
    );
    expect(result).toContain('} as const;\nexport default theme;');
```

---

We can also add the contrary expectation👇 in `Should generate the theme object` statement
```ts
expect(result).toContain('};\nexport default theme;');
```